src/main/java/com/elixr/poc/rest/response/ExceptionHandler.java

```diff
@@ -0,0 +1,41 @@
+package com.elixr.poc.rest.response;
+
+import com.elixr.poc.rest.request.UserRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+Handling the exception and sending propper error message
+ */
+@ControllerAdvice
+public class ExceptionHandler {
+    private ErrorResponse errorResponse;
+
+
+    @org.springframework.web.bind.annotation.ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<?> myMessage(MethodArgumentNotValidException methodArgumentNotValidException) {
+        return getResponseEntity(methodArgumentNotValidException);
```

Comment:
This private method call is not needed here. you can directly keep the code inside this method itself